### PR TITLE
Feature/pp9 10392 cleanup angular components

### PR DESCRIPTION
**Pull request by rjs-picturepark**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/src/app/components/application-menu/application-menu.component.scss [hunk 1]
```diff
@@ -108,4 +108,9 @@ span.label {
     position: absolute;
     width: 100%;
     bottom: 0px;
-}
```
Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/src/app/components/application-menu/application-menu.component.scss]: Do we need the extra blank lines on this file?

File: src/picturepark-sdk-v1-angular/src/app/components/application-menu/application-menu.module.ts [hunk 2]
```diff
@@ -2,10 +2,10 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { RouterModule } from '@angular/router';
 
-import { MaterialsModule } from '../../materials-module/materials-module.module';
 
 // COMPONENTS
 import { ApplicationMenuComponent } from './application-menu.component';
+import { MaterialsModule } from 'projects/picturepark-sdk-v1-angular-ui/src/lib/materials-module/materials-module.module';
```
Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/src/app/components/application-menu/application-menu.module.ts]: This should be imported from @picturepark/sdk-v1-angular-ui

Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/src/app/components/application-menu/application-menu.module.ts]: On a second note, I don't think it is a good practice to import this module from _picturepark-sdk-v1-angular-ui_.
_picturepark-sdk-v1-angular-ui_ already references _picturepark-sdk-v1-angular_, so we end up having circular references.
In my opinion, the best way is to move the Materials module to the _picturepark-sdk-v1-angular_ project or, if not possible, copy it to _picturepark-sdk-v1-angular_.

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-download-dialog/content-download-dialog.module.ts [hunk 3]
```diff
@@ -17,8 +17,5 @@ import { DialogModule } from '../dialog/dialog.module';
   exports: [
     ContentDownloadDialogComponent,
   ],
-  entryComponents: [
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-download-dialog/content-download-dialog.module.ts]: Test if entry components are breaking on run time in content portal and excel app

**Pull request merged**